admin_guide/topics/about-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='about-cluster-auto-scaler-{context}']
+= About the {product-title} auto-scaler
+
+The auto-scaler in {product-title} repeatedly checks to see how many
+pods are pending node allocation. If pods are pending allocation and
+the auto-scaler has not met its maximum capacity, then new nodes are
+continuously provisioned to accommodate the current demand. When
+demand drops and fewer nodes are required, the auto-scaler removes
+unused nodes. After you install the auto-scaler, its behavior is automatic. You
+only need to add the desired number of replicas to the deployment.
+
+In {product-title} version {product-version}, you can deploy the
+auto-scaler only on Amazon Web Services (AWS). The auto-scaler uses
+some standard AWS objects to manage your cluster size, including Auto
+Scaling groups (ASGs) and Launch Configurations (LCs).
+
+The auto-scaler uses the following assets:
+
+Auto Scaling groups:: An ASG is a logical
+representation of a set of machines. You configure an ASG with a
+minimum number of instances to run, the maximum number of instances
+that can run, and your desired number of instances to run. An ASG
+starts by launching enough instances to meet your desired capacity.
+You can configure an ASGs to start with zero instances.
+
+Launch Configurations::
+A LC is a template that an ASG uses to launch
```

Comment:
s/A LC/An LC

Reply:
> s/A LC/An LC

Seems harder (to say out loud at least). "An launch configuration" vis-a-vis "A launch configuration".

Reply:
@frobware For acronyms we use a or an based how the acronym sounds. An ell-cee (LC)

Reply:
ic

Reply:
I don't see ASG or LC in the AWS docs, so I'm expanding all the references.